src/it/java/org/checkstyle/base/AbstractItModuleTestSupport.java

```diff
@@ -218,6 +263,27 @@ protected final void verify(Configuration config, String fileName, String[] expe
                 fileName, expected, warnsExpected);
     }
 
+    /**
+     * Performs verification of the file with given file name. Uses specified configuration.
+     * Expected messages are represented by the array of strings, warning line numbers are
+     * represented by the array of integers.
+     * This implementation uses overloaded
+     * {@link AbstractItModuleTestSupport#verify(Checker, File[], String, String[], Integer...)}
+     * method inside.
+     *
+     * @param config list of configuration.
+     * @param fileName file name to verify.
+     * @param expected an array of expected messages.
+     * @param warnsExpected an array of expected warning numbers.
+     * @throws Exception if exception occurs during verification process.
+     */
+    protected final void verify(List<Configuration> config, String fileName, String[] expected,
```

Comment:
Why not just pass in the single TreeWalker configuration and use existing methods.

Reply:
Okay, haven't explored this possibility. But @rnveach  wont that include all the configuration inside `TreeWalker.` present in google_checks.xml
In that case how can I get the required modules?

Reply:
I referring to doing the following:

verify(createTreeWalkerConfig(config))

You created a version where we can build TreeWalker config from multiple configs. Since you are now passing in a single config, the existing areas should already cover this.

Reply:
@rnveach made the changes. 

Added Input files this time to test the changes in #7542.
Please whether these are the changes you are asking for.